USAGE.md

```diff
@@ -28,39 +25,13 @@ For more information on various event types emitted by the node, visit the [Moni
 It is possible to monitor the Sidecar event stream using *cURL*, depending on how the HOST and PORT are configured.
 
 ```json
-curl -s http://<HOST:PORT>/events/<TYPE>
+curl -s http://<HOST:PORT>/events
 ```
 
 - `HOST` - The IP address where the Sidecar is running
 - `PORT` - The port number where the Sidecar emits events
 - `TYPE` - The type of event emitted
 
-Given this [example configuration](./resources/example_configs/EXAMPLE_NODE_CONFIG.toml), here are the commands for each endpoint:
-
-- **Deploy events:** 
-
-    ```json
-    curl -sN http://127.0.0.1:19999/events/deploys
-    ```
-
-- **Finality Signature events:** 
-
-    ```json
-    curl -sN http://127.0.0.1:19999/events/sigs
-    ```
-
-- **Main events:** 
-
-    ```json
-    curl -sN http://127.0.0.1:19999/events/main
-    ```
-
-- **Sidecar-generated events:** 
-
-    ```json
-    curl -sN http://127.0.0.1:19999/events/sidecar
-    ```
-
```

Comment:
I'd like to add 2 examples:

Given this [example configuration](./resources/example_configs/EXAMPLE_NODE_CONFIG.toml), here are the commands for each endpoint:

- **Node events:** 

    ```json
    curl -sN http://127.0.0.1:19999/events
    ```

- **Sidecar-generated events:** 

    ```json
    curl -sN http://127.0.0.1:19999/events/sidecar
    ```